themes/default/content/blog/introducing-azure-native-v2/index.md

```diff
@@ -0,0 +1,113 @@
+
+If you would like to upgrade to 2.x, but continue using the same Azure API version that was used in your 1.x, you can do that by specifying the compatible explicit version available in the 2.x provider. Shown below are examples of how that change might look.
+
+{{< chooser language "typescript,python,csharp,go,yaml" >}}
+
+{{% choosable language typescript %}}
+
+```typescript
+- import { EventSubscription } from "@pulumi/azure-native/eventgrid";
++ import { EventSubscription } from "@pulumi/azure-native/eventgrid/v20200601";
+```
+
+{{% /choosable %}}
+{{% choosable language python %}}
+
+```python
+- from pulumi_azure_native import eventgrid
++ from pulumi_azure_native.eventgrid import v20200601 as eventgrid20200601
+```
+
+{{% /choosable %}}
+{{% choosable language csharp %}}
+
+```csharp
+- using EventGrid = Pulumi.AzureNative.EventGrid;
++ using EventGrid20200601 = Pulumi.AzureNative.EventGrid.V20200601;
+```
+
+{{% /choosable %}}
+{{% choosable language go %}}
+
+```go
+- import eventgrid "github.com/pulumi/pulumi-azure-native-sdk/eventgrid/v2"
++ import eventgrid20200601 "github.com/pulumi/pulumi-azure-native-sdk/eventgrid/v2/v20200601”
+```
+
+{{% /choosable %}}
+{{% choosable language yaml %}}
+
+```yaml
+- type: azure-native:eventgrid:Topic
++ type: azure-native:eventgrid/v20200601:Topic
+```
+
+{{% /choosable %}}
+{{< /chooser >}}
+
+## Expanded Top-Level Default Module Coverage
+
+We're excited to expand our top-level default version coverage by 226 resources, 274 functions, and 5526 input properties in this release, bringing the total to 1,637, 2,015, and 30,652 respectively. With this extensive collection of modules at your disposal, you can easily provision and manage the full range of Azure services using Pulumi. Whether you're working with virtual machines, databases, networking, or any other Azure service, the Azure Native Provider has you covered.
+
+![provider-coverage-by-version](./azurev2-provider-coverage.png)
+
+## Resolved Title Case Inconsistency
+
+In our commitment to excellence, we have [resolved title case inconsistencies](https://github.com/pulumi/pulumi-azure-native/pull/2366) across module types and properties in the Azure Native Provider. All previous module names have been aliased, so no additional modifications are required for existing Pulumi programs. This enhancement ensures a unified and predictable experience while switching between Azure and Pulumi module references, making it easier for you to navigate and understand the provider's capabilities.
+
+## Simplified User Assigned Identity Inputs
+
+We have simplified the representation of user assigned identity inputs by recognizing them as string arrays. This change provides a more intuitive and clear approach when configuring user assigned identities for your Azure modules. You can now reference these as `UserAssignedIdentities: [exampleUserAssignedIdentity.id]` without jumping through any hoops.
```

Comment:
Do we have any larger examples of this?  Even something in the registry we could link to?  Might be nice just to provide a little more context.  I recall this was one of the most requested issues for the Azure Native provider.

Reply:
Added. Will add a chooser later to include other languages.